e2e/cypress/integration/messaging/center_channel_rhs_overlap_spec.js

```diff
@@ -49,5 +81,810 @@ describe('Messaging', () => {
+
+            // * Edit post modal is still visible after typing ENTER
+            cy.get('#editPostModal').should('be.visible');
+
+            // [24] Press CTRL+ENTER
+            cy.typeCmdOrCtrlForEdit().type('{enter}').wait(TIMEOUTS.HALF_SEC);
+        });
+
+        // * Check that the edited message has been posted
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`).should('contain', messageWithCodeblockTextOnly2);
+        });
+
+        // # [25] Post message with quotes with caret in the middle
+        cy.get('#post_textbox').should('be.visible').clear().type(`${messageWithCodeblockIncomplete3}{leftArrow}{leftArrow}{leftArrow}{enter}`).wait(TIMEOUTS.HALF_SEC);
+
+        // * Check that the message has not been posted
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`).should('not.contain', messageWithCodeblockTextOnly3);
+        });
+
+        // # [26] Press CTRL+ENTER
+        // * Post message again (previous one is broken)
+        cy.get('#post_textbox').should('be.visible').clear().type(`${messageWithCodeblockIncomplete3}{leftArrow}{leftArrow}{leftArrow}`).wait(TIMEOUTS.HALF_SEC);
+
+        cy.typeCmdOrCtrl().type('{enter}');
+
+        // * Check that the message has been posted
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`).should('contain', messageWithCodeblockTextOnly3);
+        });
+
+        // # [27] Edit previous post
+        cy.getLastPostId().then(() => {
+            cy.get('#post_textbox').type('{uparrow}');
+
+            // * Edit post modal should appear
+            cy.get('#editPostModal').should('be.visible');
+
+            // * Update the post message and type ENTER
+            cy.get('#edit_textbox').invoke('val', '').type(`${messageWithCodeblockIncomplete3}{leftArrow}{leftArrow}{leftArrow}`).wait(TIMEOUTS.HALF_SEC);
+            cy.get('#edit_textbox').type('{enter}');
+
+            // * Edit post modal is still visible after typing ENTER
+            cy.get('#editPostModal').should('be.visible');
+
+            // [28] Press CTRL+ENTER
+            // * Post message again (previous one is broken)
+            cy.get('#edit_textbox').invoke('val', '').type(`${messageWithCodeblockIncomplete3}{leftArrow}{leftArrow}{leftArrow}`).wait(TIMEOUTS.HALF_SEC);
+            cy.typeCmdOrCtrlForEdit().type('{enter}').wait(TIMEOUTS.HALF_SEC);
+        });
+
+        // * Check that the message has been posted
+        // Bug? - when clocking CTRL+ENTER from edit box, the code block does not appear in center
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`).should('contain', messageWithCodeblockTextOnly3);
+        });
+    });
+
+    it('MM-T712_3 Editing a post and hitting <enter> in code block saves post automatically instead of adding newline', () => {
```

Comment:
I extracted the other case into its own test case. Now as [MM-T3449](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T3449)
```suggestion
    it('MM-T3449 Editing a post with Ctrl+Enter off for code blocks configured', () => {
```